src/main/resources/assets/immersiveengineering/lang/en_us.json

```diff
@@ -731,15 +731,19 @@
   "effect.immersiveengineering.concrete_feet": "Concrete Feet",
   "effect.immersiveengineering.flashed": "Flashed",
   "manual.immersiveengineering.manual": "Engineer's Manual",
-  "manual.immersiveengineering.update": "UPDATE NEWS!",
-  "manual.immersiveengineering.general": "Overview and Resources",
-  "manual.immersiveengineering.early_machines": "Workbenches & Furnaces",
+  "manual.immersiveengineering.updates_and_introduction": "Updates & Introduction!!",
+  "manual.immersiveengineering.introduction": "Introduction",
```

Comment:
Why is there two categories for Indtroduction?
"Update News" should stay as they are please, as they are exclusively auto-generated.